about_bootstrap.md

```diff
@@ -0,0 +1,29 @@
+## What you will learn in this section
+This section will basically help you to know what is bootstrap and why you should learn it.
+
+## About bootstrap
+Bootstrap, originally named Twitter Blueprint, was developed by Mark Otto and Jacob Thornton at Twitter as a framework to encourage consistency across internal tools. Here we will talk about why Bootstrap has capturted the developers attraction:
+
+We always think of investment and how to save money, talking of Bootstrap it is free and open-source frontend framework for designing website or web based applications. But I agree there would a question in your mind what does this bootstrap contain and how it is used?
+
+We will learn how it is used in this tutorial but first let me discuss what it contains. Bootstrap contains HTML, CSS based designed template for verious components like button, pagination, progress bar, different icon and Javascript extension.
+
+But it raises another question why we should learn it?
+
+## Why you should learn Bootstrap
+
+### Features
+Bootstrap is compatible with latest version of the Google Chrome, Firefox, Internet Explorer, Safari and Opera browser. It gives you responsive website where your website would scale according with the size of the device. Earlier developer have to write different lines of code for different devices. It is just a first step and you guess how powerful is the Bootstrap.
+
+### Free and open-source
+As I already discussed with you Bootstrap is free and open-source which make it developer friendly and you can even get the code help or supoort through different forum which work openly in the contribution towards open-source. You can also contribute to the Bootstrap  some of your codes and can gain your name in and around Globe. 
```

Comment:
Remove - You can also contribute to the Bootstrap  some of your codes and can gain your name in and around Globe. 

And rephrase the paragraph.